frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -48,11 +48,12 @@ export function compareActivities(a, b) {
 /**
  * Get the field of field name `fieldName` from `activity  or the default value.
  * 
- * @param {Object} activity 
- * @param {string} fieldName 
- * @param defaultValue 
- * @param prefix The prefix to put before a returned value if the field exists.
- * @returns `activity[fieldName]` if possible, else `defaultValue`.
+ * @param {Object} activity Activity to get field from.
+ * @param {string} fieldName Name of the field in the activity to get. 
+ * @param defaultValue Default value to return if activity[fieldName] can't be found. 
+ * Can be any type.
```

Comment:
Can you use {*} here to represent that? https://devhints.io/jsdoc
Or if it's not really any type, but rather just 1 of several, you can specify that too (see link for that as well).